src/index.js

```diff
@@ -1,5 +1,8 @@
+// @flow
 /**
  * React Flip Move
  * (c) 2016-present Joshua Comeau
  */
-module.exports = require('./FlipMove');
+import FlipMove from './FlipMove';
+
+export default FlipMove;
```

Comment:
Good call on skipping `export default from`, didn't realize it wasn't a part of the initial module syntax.

And yeah if it seems to work, I'm alright with it. I remember that there was a bug with something, but it's been a year, and a bunch of other libraries use the same build system without issues, so -shrugs-.